test/pyseir_end_to_end_test.py

```diff
@@ -12,19 +15,22 @@
 @pytest.mark.slow
 def test_pyseir_end_to_end_idaho():
     # This covers a lot of edge cases.
-    # cli._run_all(state='Idaho')
-    cli._build_all_for_states(states=["ID"], fips="16001")
-    path = get_run_artifact_path("16001", RunArtifact.WEB_UI_RESULT).replace(
-        "__INTERVENTION_IDX__", "2"
-    )
-    path = pathlib.Path(path)
-    assert path.exists()
-    output = CANPyseirLocationOutput.load_from_path(path)
-    data = output.data
-    with_values = data[schema.RT_INDICATOR].dropna()
-    assert len(with_values) > 10
-    assert (with_values > 0).all()
-    assert (with_values < 6).all()
+
+    with temppathlib.TemporaryDirectory() as tmp_dir:
+        with unittest.mock.patch("pyseir.utils.OUTPUT_DIR", tmp_dir.path):
```

Comment:
This would be handy to make an environment variable I think as well

Reply:
Perhaps, though I haven't been motivated for that so far...